docs/upgrading.rst

```diff
@@ -0,0 +1,447 @@
+Upgrading to Newer Releases
+===========================
+
+This section documents migration paths to new releases.
+
+Upgrading to 6.0
+++++++++++++++++
+
+In webargs 6, the way that data is loaded from various locations and passed to
+your schemas changed significantly. This may require you to change several
+different parts of your webargs usage.
+
+Multiple locations are no longer supported in a single call
+-----------------------------------------------------------
+
+And only JSON data is parsed by default.
```

Comment:
The default location is JSON/body.

Reply:
I take it this is a suggested rewording? I'll go with it. It's shorter than what I had.